dashboards/rabbitmq/rabbitmq-prometheus.json

```diff
@@ -0,0 +1,889 @@
+{
+  "category": "CUSTOM",
+  "displayName": "RabbitMQ Overview",
```

Comment:
Display name should be "RabbitMQ Prometheus Overview" to disambiguate from the other overview dashboard.